edualgo/graph_algorithms/Dijkstra_algorithms.py

```diff
@@ -0,0 +1,112 @@
+from .__init__ import print_msg_box
+
+import sys
+
+
+def dijkstra_algorithm(graph, hint=False):
+
+    # Providing the graph
+    size = int(input())  # Size of the Vertices & Edges
```

Comment:
When the user has passed graph as parameter already why to take input explicitly for size ?

Reply:
then how we will take the input for vertices and edges ?